tests/test_pacer.py

```diff
@@ -634,6 +634,13 @@ def test_district_court_dockets(self):
                                  'district')
         self.run_parsers_on_path(path_root)
 
+    def test_district_court_dockets(self):
```

Comment:
This method has the same name as the one before?

Reply:
Err...huh. So it does. I'm not sure what to make of the fact that that seemed to perform fine.